components/serializer.rst

```diff
@@ -1254,6 +1247,52 @@ These are the options available:
 ``remove_empty_tags``
     If set to true, removes all empty tags in the generated XML.
 
+The CSV Encoder
+---------------
+
+This encoder transforms arrays into XML and vice versa.
```

Comment:
Xml?

Reply:
Yeah, "text is based on XML's part" meant ctrl+c ctrl+v ;) Fix is on the way